app/models/customer.py

```diff
@@ -0,0 +1,33 @@
+from flask import current_app
+from app import db
+from datetime import datetime
+
+class Customer(db.Model):
+    customer_id = db.Column(db.Integer, primary_key=True, autoincrement=True)
+    name =  db.Column(db.String)
+    postal_code = db.Column(db.String)
+    phone = db.Column(db.String)
+    register_at = db.Column(db.DateTime, default=datetime.utcnow())
+    videos_checked_out_count = db.Column(db.Integer, default=0)
+    rentals = db.relationship('Rental', backref='customer', lazy=True)
+
+
+    def to_json(self):
+        if self.videos_checked_out_count:
+            return{
+                "id": self.customer_id,
+                "name": self.name,
+                "phone": self.phone,
+                "postal_code": self.postal_code,
+                "registered_at": self.register_at,
+                "videos_checked_out_count": self.videos_checked_out_count
```

Comment:
Using the relationship the `videos_checked_out_count` is just the length of the `rentals` that haven't been checked in:

```suggestion
            "videos_checked_out_count": len([rental for rental in self.rentals if not rental.check_in_date])
```

This also means you don't need to special case things since if no rentals exist it just computes `0`.